appengine/typescript/package.json

```diff
@@ -9,26 +9,27 @@
     "node": "16.x.x"
   },
   "scripts": {
-    "prepare": "npm run gcp-build",
-    "pretest": "npm run gcp-build",
+    "prepare": "npm run compile",
+    "pretest": "npm run compile",
     "test": "mocha test/*.test.js --exit",
-    "posttest": "npm run lint",
-    "lint": "tslint -p .",
+    "lint": "gts lint",
     "start": "node ./index.js",
-    "gcp-build": "tsc -p .",
```

Comment:
Was this removed intentionally, the documentation still refers to this as an option for running an custom build script: https://cloud.google.com/appengine/docs/standard/nodejs/running-custom-build-step#example

Reply:
I am pretty sure this was a mistake. The docs are correct.

Reply:
This change actually broke the ability to deploy to GAE. It now fails in the build phase with an error like:

```
ERROR: (gcloud.app.deploy) Error Response: [9] Cloud build cdec98ca-e4aa-4a97-bbfc-d5843c7ff8a3 status: FAILURE
> appengine-typescript@0.0.1 prepare
> npm run compile


> appengine-typescript@0.0.1 compile
> tsc -p .

sh: 1: tsc: not found
npm ERR! code 127
npm ERR! path /workspace
npm ERR! command failed
npm ERR! command sh -c -- npm run compile

npm ERR! A complete log of this run can be found in:
```

This is because the `devDependencies are not installed when the prepare script is run. The `gcp-build` script fixes this.

Reply:
This change has been reverted, and a test with `npm run deploy` was successful.

Reply:
FYI, we are currently working on a change to run `npm run build` by default when deploying to AppEngine, because this `gcp-build`thing is not very intuitive. 

cc @anniefu